20151014/python/Alex_James_Traffic_Flow_Python_Project.py

```diff
@@ -0,0 +1,180 @@
+#---------------------------Import-------------------------------------#
+
+import numpy as np
+from physics import *
+from _random import *
+import pylab
+
+#---------------------------Initial Set Up-----------------------------#
+
+N=10                                    #Fixed Number of cars
+
+V_max=5                                 #Speed limit
+V_trans=3                               #transition v for prob
+P_fast=0.02                             #brake prob for fast car
+P_slow=0.25                             #brake prob for slow car
+
+L_max=20*N                              #Max no. of cells in road
+L_start=1*N                             #Min no. of cells in road
+L_int=1                                 #L interval
+
+t_count=100                             #Time to start counting
+t_max=1000                              #End time of simulation
+N_exp=1                                 #Number of expiriments to average
+
+#-------------------------Arrays For Plotting---------------------------#
+
+L_array=[]               #just making the necessary arrays for plotting           
+density_array=[]
+all_counts=[]
+counts_to_average=np.zeros(((L_max)-L_start)/L_int)
+counts_averaged=[]
+v_to_average=np.zeros(((L_max)-L_start)/L_int)

```

Comment:
To avoid repeating ` ((L_max)-L_start)/L_int)` I would define it before with an understandable name, eg. `steps`